### ISSUE TITLE
Billing amount is updated after reloading the page

### ISSUE THREAD

**Issue by diclee** [koding/koding#9348]
## Current Behavior

![gif](http://g.recordit.co/zOWOzXjffK.gif)
http://recordit.co/zOWOzXjffK

**Comment by usirin** [comment]
Self note: listen for new member event for team.

**Comment by usirin** [comment]
This is gonna be solved with #9965

**Comment by usirin** [comment]
@diclee we don't have a realtime event coming from backend for us to track the realtime updates of counted user changes, because those are counted dynamically by the backend. 

I will instead make sure to fetch the latest version of the data (the data comes only after a full reload right now) whenever user tries to open the billing pane to improve a little bit.

In the meantime @cihangir do you have anything you want to add here?

**Comment by cihangir** [comment]
@usirin listening to member added events will improve a lot 👍 

Do you guys think we should send a realtime event when an already existent member logins in to koding for the first time in a month to reflect the latest status? seems nice to have to me

**Comment by gokmen** [comment]
It would be nice to see who is taking into account as well, it happened to me once that I've invited someone to my team and one day after I kicked that user. Payment showed me 2 users and a link goes to members list which only has one user in it (since I've kicked and removed that user already).  It would be nice who is exactly those 2 users were.

**Comment by cihangir** [comment]
We store the active users' list, we can show them 👌
